js/script.js

```diff
@@ -0,0 +1,75 @@
+const searchButton = document
+  .getElementById("searchBtn")
+  .addEventListener("click", searchData);
+const userInput = document.getElementById("input");
+const searchedList = document.getElementById("listGroup");
+const loadingSpinner = document.getElementById("spinner");
+
+function searchData(e) {
+  e.preventDefault();
+  getSearchData(
+    "https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/search?query=" +
```

Comment:
use baseUrl bar to store 'https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/'. DRY